Feature/expiration date preference (#897)
mercadopago/devsite-docs: guides/online-payments/checkout-pro/latest/configurations.en.md

```
]]]

----[mla, mlb, mco]----

## Expiration date of cash payment

> The default expiration date for cash payments is 30 days. If you want, you can change this date by sending the `date_of_expiration` field in the preference creation request. The configured date must be between 1 and 30 days from the preference creation date.

Reviewer: ```suggestion
The default expiration date can vary between 5 to 30 days depending on the means of payment you are using.. If you want, you can change this date by sending the `date_of_expiration` field in the preference creation request. The configured date must be between 5 and 30 days from the preference creation date.
```

Reviewer 2: ```suggestion
By default, the due date of a cash payment can vary between 5 and 30 days depending on the payment method you are using. If you want, you can change this date by sending the `date_of_expiration` field in the preference creation request. The configured date must be between 1 and 30 days from the preference creation date.
```

> The default expiration date for cash payments is 30 days. If you want, you can change this date by sending the `date_of_expiration` field in the preference creation request. The configured date must be between 1 and 30 days from the preference creation date.

Reviewer 2: ```suggestion
If you want, you can change the default due date of a cash payment by sending the `date_of_expiration` field in the preference creation request. The configured date must be between 1 and 30 days from the preference creation date.
```

[[[
```json
===
The date uses the ISO 8601 format: yyyy-MM-dd'T'HH:mm:ssz
===

> "date_of_expiration": "2020-05-30T23:59:59.000-04:00";

Reviewer 2: ```suggestion
"date_of_expiration": "2020-05-30T23:59:59.000-04:00"
```

```
]]]

The deadline for approval of the cash payment is between 1 and 2 working days according to the payment method. Therefore, we recommend that you set the due date with at least 3 days to ensure that payment is made.

> Check [credit times by payment method](https://www.mercadopago[FAKER][URL][DOMAIN]/ayuda/Medios-de-pago-y-acreditaci-n_221) when configuring.

Reviewer 2: ```suggestion
Check [credit times by payment method](https://www.mercadopago[FAKER][URL][DOMAIN]/ayuda/_221) when configuring.
```
